src/ethereum/vm/stack.py

```diff
@@ -64,3 +64,62 @@ def push(stack: List[U256], value: U256) -> None:
         raise StackOverflowError
 
     return stack.append(value)
+
+
+def push_multiple(stack: List[U256], values: Sequence[U256]) -> None:
```

Comment:
I think the readability of multiple individual `push` calls over one `push_multiple` trumps performance in this case, unless you've measured a significant difference in speed?